src/Initializer.ts

```diff
@@ -56,7 +62,19 @@ export abstract class Initializer {
 			}
 		});
 
-		await this.doStep("Installing @rbxts/types", () => cmd("npm", ["i", "-D", "@rbxts/types"]));
+    switch (dependencyMode) {
+      case DependencyMode.npm: {
+        await this.doStep("Installing @rbxts/types", () => cmd("npm", ["i", "-D", "@rbxts/types"]));
+        break;
+      }
+      case DependencyMode.pnpm: {
+        await this.doStep("Installing @rbxts/types", () => cmd("pnpm", ["i", "-D", "@rbxts/types"]));
+        break;
+      }
+      case DependencyMode.off: {
+        break;
+      }
+    }
```

Comment:
Since that `npm` and `pnpm` use the same interface, we can join them and shave off a couple lines of code...

```suggestion
		switch (dependencyMode) {
			case DependencyMode.npm:
			case DependencyMode.pnpm: {
				await this.doStep("Installing @rbxts/types", () => cmd(dependencyMode, ["i", "-D", "@rbxts/types"]));
				break;
			}
			case DependencyMode.yarn: {
				await this.doStep("Installing @rbxts/types", () => cmd("yarn", ["add", "--dev", "@rbxts/types"]));
				break;
			}
			case DependencyMode.off: {
				break;
			}
		}
```